front/cypress/integration/content_builder/components/iframe_component.ts

```diff
@@ -0,0 +1,120 @@
+import { randomString } from '../../../support/commands';
+
+describe('Content builder Iframe component', () => {
+  let projectId = '';
+  let projectSlug = '';
+
+  const getIframeBody = () => {
+    return cy
+      .get('iframe')
+      .its('0.contentDocument')
+      .should('exist')
+      .its('body')
+      .should('not.be.undefined')
+      .then(cy.wrap);
+  };
+
+  before(() => {
+    cy.setAdminLoginCookie();
+    cy.getAuthUser().then((user) => {
+      const projectTitle = randomString();
+      const projectDescriptionPreview = randomString();
+      const projectDescription = 'Original project description.';
+      const userId = user.body.data.id;
+
+      cy.apiCreateProject({
+        type: 'continuous',
+        title: projectTitle,
+        descriptionPreview: projectDescriptionPreview,
+        description: projectDescription,
+        publicationStatus: 'published',
+        participationMethod: 'ideation',
+        assigneeId: userId,
+      }).then((project) => {
+        projectId = project.body.data.id;
+        projectSlug = projectTitle;
+        cy.visit(`/admin/projects/${projectId}/description`);
+        cy.get('#e2e-toggle-enable-content-builder')
+          .find('input')
+          .click({ force: true });
+      });
+    });
+  });
+
+  beforeEach(() => {
+    cy.setAdminLoginCookie();
+  });
+
+  after(() => {
+    cy.apiRemoveProject(projectId);
+  });
+
+  it('handles Iframe component correctly', () => {
+    // Add iframe with valid url
+    cy.visit(`/admin/content-builder/projects/${projectId}/description`);
+    cy.get('#e2e-draggable-iframe').dragAndDrop('#e2e-content-builder-frame', {
+      position: 'inside',
+    });
+    cy.get('#e2e-iframe-url').type(
+      // Typeform survey created in CitizenLab Methods Squad workspace specifically for e2e
+      'https://citizenlabco.typeform.com/to/cZtXQzTf'
+    );
```

Comment:
I wasn't quite sure what URL to test with, as I wanted to make sure the iframe content renders correctly. I figured a specific typeform survey in our workspace would be ok?